src/auth/auth.service.ts

```
@@ -429,4 +429,22 @@ export class AuthService {
       }
     }
   }
+
+  // 귀여운 랜덤 이름 만드는 함수
+  generateRandomUserName(): string {
+    const adjectives = [
+      '귀여운', '싱그러운', '느긋한', '화려한', '조용한', '햇살 가득한', '활기찬', '수줍은', '포근한', '청조한',
```

ESLint

[error] Replace `·'싱그러운',·'느긋한',·'화려한',·'조용한',·'햇살·가득한',·'활기찬',·'수줍은',·'포근한',` with `⏎······'싱그러운',⏎······'느긋한',⏎······'화려한',⏎······'조용한',⏎······'햇살·가득한',⏎······'활기찬',⏎······'수줍은',⏎······'포근한',⏎·····`

(prettier/prettier)